packages/FormElement/stories/FormElement.examples.stories.js

```diff
@@ -22,69 +23,89 @@ export default {
 
 export const accessibilityExample = () => (
   <ExampleStory storyName="Accessibility" fileName="examples/AccessibilityExample.js">
+    <Tagline>Form Element with instructions component.</Tagline>
```

Comment:
You can more simply achieve this by using the `tagline` prop of the `<ExampleStory>`